site/src/07-22-manifest.json

```diff
@@ -0,0 +1,4038 @@
+{
```

Comment:
Would it be possible to publish a manifest like this (or at least just a list of file names) alongside each overture release?

Reply:
Ah I realize this is similar to @H-Plus-Time's comment. It seems like this could either go in this repo to benefit this one site, overture releases to benefit all tools that work with overture data, or even into the geoparquet spec to benefit other datasets as well. Seems like it helps bridge the gap from parquet's initial Hadoop/mapreduce roots to more modern use cases like this site.

Reply:
@msbarry yes, that's the eventual goal of this sort of work.  I saw that you'd also commented on https://github.com/OvertureMaps/data/issues/25 from last year and understand your sentiments. 

I've put the beginnings of a manifest file creation tool in my own personal github repo at https://github.com/Bonkles/manifest-generator to try and alleviate this kind of concern. Definitely interested in any other ideas around info the manifest should contain.

Reply:
Nice! I think the bare minimum (not overture specific) would be a list of: relative path, file size, and geoparquet metadata (or a subset like bbox, geometry types if the entire metadata is too big). Anything overture-specific beyond that could be nice to limit processing but not strictly necessary.